include/alpaka/acc/AccOmp5.hpp

```diff
@@ -147,36 +139,45 @@ namespace alpaka
         namespace traits
         {
             //#############################################################################
-            //! The CPU OpenMP 4.0 accelerator accelerator type trait specialization.
+            //! The OpenMP 5.0 accelerator accelerator type trait specialization.
             template<
                 typename TDim,
                 typename TIdx>
             struct AccType<
-                acc::AccCpuOmp4<TDim, TIdx>>
+                acc::AccOmp5<TDim, TIdx>>
             {
-                using type = acc::AccCpuOmp4<TDim, TIdx>;
+                using type = acc::AccOmp5<TDim, TIdx>;
             };
             //#############################################################################
-            //! The CPU OpenMP 4.0 accelerator device properties get trait specialization.
+            //! The OpenMP 5.0 accelerator device properties get trait specialization.
             template<
                 typename TDim,
                 typename TIdx>
             struct GetAccDevProps<
-                acc::AccCpuOmp4<TDim, TIdx>>
+                acc::AccOmp5<TDim, TIdx>>
             {
                 //-----------------------------------------------------------------------------
                 ALPAKA_FN_HOST static auto getAccDevProps(
-                    dev::DevCpu const & dev)
+                    dev::DevOmp5 const & dev)
                 -> acc::AccDevProps<TDim, TIdx>
                 {
+                    alpaka::ignore_unused(dev);
+
+#if defined(ALPAKA_OFFLOAD_MAX_BLOCK_SIZE) && ALPAKA_OFFLOAD_MAX_BLOCK_SIZE>0
+                    auto const blockThreadCount = ALPAKA_OFFLOAD_MAX_BLOCK_SIZE;
```

Comment:
Is `ALPAKA_OFFLOAD_MAX_BLOCK_SIZE` meant to only reduce the `blockThreadCount` when there are too many available?
Should we then do `std::min(::omp_get_max_threads(), ALPAKA_OFFLOAD_MAX_BLOCK_SIZE)`?

Reply:
Yes, this is sensible.